torch/_torch_docs.py

```diff
@@ -4530,6 +4530,11 @@ def merge_dicts(*dicts):
 
 Returns the median value of all elements in the :attr:`input` tensor.
 
+.. note::
```

Comment:
This note is a little wonky because median, like other reductions, is documented twice. Worse, it's really two different functions: median and median_with_indices, which is definitely an API bug that we shouldn't try to address in this PR. Unfortunately, it means you probably need to duplicate this note for both versions of the documentation.